autosklearn/pipeline/components/feature_preprocessing/nystroem_sampler.py

```diff
@@ -94,7 +95,7 @@ def get_properties(dataset_properties=None):
         }
 
     @staticmethod
-    def get_hyperparameter_search_space(dataset_properties=None):
+    def get_hyperparameter_search_space(feat_type=None, dataset_properties=None):
```

Comment:
Add annotanies here ? dataset_properties also has no annotations

Reply:
@mfeurer

Reply:
Yes, please. You could also add the annotation for `dataset_properties`. They should be importable, too.